frame/pallet-xvm/src/lib.rs

```diff
@@ -80,6 +80,12 @@ pub struct XvmCallOk {
     consumed_weight: u64,
 }
 
+impl XvmCallOk {
+    pub fn output(&self) -> &Vec<u8> {
```

Comment:
Just wondering, how come you're using direct Vec reference instead of slice?

I thought that in this context, `slice` would be more idiomatic.

Reply:
You're right that slice would be more idiomatic. 

Originally I did `&Vec` because currently we have `EvmData` implementation for `Vec` only and not for slice, so to use `output` in `EvmDataWriter` I need either a clone of vector or a reference. But then I found out that I need to use `Bytes(Vec<u8>)` to do compact byte encoding.

Anyways, good catch, will change it to slice.